bin/format

```diff
@@ -64,6 +64,7 @@ docker_clang_format() {
         process_arg "$arg"
     done | xargs -0 \
       docker run \
+      --platform linux/amd64 \
```

Comment:
Is this change necessary? IIRC docker would use the right platform depending on the host platform.

Reply:
fixed in 382e80faa81e353100104c43834c17c32ee0412b